AirCasting/Extensions/Sequence.swift

```diff
@@ -0,0 +1,21 @@
+// Created by Lunar on 30/08/2022.
```

Comment:
just one more thing - please rename this file to be of `Sequence+MapWithNext.swift`. This is a standard convention for files containing extensions. Alternatively we can just make it `Sequence+Helpers.swift` so in the future we can dump all convenience extensions in a single file